components/bugzilla.py

```diff
@@ -82,9 +82,9 @@ def __init__(self, config):
                 assert ('url' in self.config) or (self.config['General']['env'] in ["dev", "prod"]), "No bugzilla url provided, and unknown operating environment"
 
     @logEntryExit
-    def file_bug(self, library, new_release_version):
-        summary = "Update %s to new version %s" % (
-            library.shortname, new_release_version)
+    def file_bug(self, library, new_release_version, release_timestamp):
+        summary = "Update %s to new version %s from %s" % (
```

Comment:
Can do something like

```
try:
   d = datetime.parse(timestamp, "blah"")
   timestamp_str = d.format("...")
except:
   timestamp_str = timestamp
```

So we can transform "2020-08-21T15:13:49.000+02:00" to "2020-08-21 15:13:49"?

Reply:
I went with using `python-dateutil` since we don't know if the format coming from mach vendor will be the same for github, gitlab, the videolan repo, etc., and this should do a best guess or just emplace as-is if we can't determine the date time string